osu.Game/Rulesets/Objects/BezierConverter.cs

```diff
@@ -87,6 +87,11 @@ public static List<Vector2> ConvertToLegacyBezier(IList<PathControlPoint> contro
 
                         break;
 
+                    case PathType.BSpline:
+                        result.AddRange(from segment in ConvertBSplineToBezierAnchors(segmentVertices) from v in segment select v + position);
```

Comment:
Ew it's the fake sql syntax. I'll look away because it's already being used elsewhere but I kinda hate it.